.prettierrc

```diff
@@ -1,4 +1,6 @@
 {
   "singleQuote": true,
-  "printWidth": 120
+  "printWidth": 120,
+  "trailingComma": "none",
```

Comment:
I kind of prefer trailingComma, reduces merge conflicts etc.

Reply:
Probably best to keep it as is and raise another PR enabling trailing comma

Reply:
I was originally aiming for lowest impact changes but I agree with the reasoning for trailing commas.  I'm happy to remove this and run prettier across the project to update.